compose.yaml

```diff
@@ -49,20 +51,39 @@ services:
   ctakes-covid:
     extends: ctakes-covid-base
     profiles:
-      - etl-support
-      - etl-support-gpu
+      - covid-symptom
+      - covid-symptom-gpu
```

Comment:
OK notable change number 1: how do you feel about this direction?

I think I mentioned wanting to toy with this - not a global "run all the services I might ever need" but rather a more targeted "run the services for the studies I'm caring about right now".

It's a more complicated user story, but I feel like the "run everything" approach quickly falls over as we add studies.

You had mentioned anticipating moving to a more comprehensive meta-runner like Kubernetes or something yeah? I'm not opposed, but in the meantime, I'm thinking something this could be a happy medium.

Reply:
yeah i don't hate this given the addition of a new task, and also that folks may not often be running this now.